inference-engine/src/mkldnn_plugin/memory_desc/dnnl_blocked_memory_desc.cpp

```diff
@@ -718,6 +717,8 @@ size_t DnnlBlockedMemoryDesc::getMaxMemSize() const {
 }
 
 size_t DnnlBlockedMemoryDesc::getPaddedElementsCount() const {
+    if (std::any_of(blockedDims.begin(), blockedDims.end(), [](Dim dim) { return dim == Shape::UNDEFINED_DIM; }))
```

Comment:
it's better to call `getBlockDims()`
and use result of this operation into `std::any_of`

because in further for `DnnlBlockedMemoryDesc` blocked params maybe not filled

same for `serializeFormat()`